modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoverySystemViews.java

```diff
@@ -65,6 +66,7 @@ static SystemView<?> createLocalPartitionStatesSystemView(DisasterRecoveryManage
                 .addColumn("TABLE_NAME", STRING, state -> state.state.tableName)
                 .addColumn("PARTITION_ID", INT32, state -> state.state.partitionId)
                 .addColumn("STATE", STRING, state -> state.state.state.name())
+                .addColumn("ESTIMATED_SIZE", INT64, state -> state.state.estimatedSize)
```

Comment:
I know that everywhere in our code it's called `estimated size`, but how about inventing a more descriptive term for the user-facing places like this one, like 'estimatedRows'? A problem with `estimatedSize` is that it's not clear in what units it is: maybe it's rows, maybe it's bytes. 'estimatedRows' would allow to remove this ambiguity.